litecommands-annotations/test/dev/rollczi/litecommands/annotations/argument/resolver/standard/InstantArgumentTest.java

```diff
@@ -25,22 +28,28 @@ Instant test(@Arg Instant instant) {
     @Test
     void test() {
         platform.execute("test 2023-10-13 11:20:00")
-            .assertSuccess(Instant.from(DateTimeFormatter
-                .ofPattern("yyyy-MM-dd HH:mm:ss")
-                .withZone(ZoneOffset.UTC)
-                .parse("2023-10-13 11:20:00")
-            ));
+            .assertSuccess(LocalDateTime.of(2023, 10, 13, 11, 20, 0).toInstant(ZoneOffset.UTC));
     }
 
     @Test
     void testInvalid() {
-        platform.execute("test 1acab")
+        platform.execute("test invalid")
+            .assertFailure();
+
+        platform.execute("test invalid 11:20:00")
+            .assertFailure();
+
+        platform.execute("test 2023-10-13 invalid")
             .assertFailure();
     }
 
     @Test
     void testSuggestions() {
-        platform.suggest("test ").getSuggestions().stream()
+        Collection<Suggestion> suggestions = platform.suggest("test ").getSuggestions();
+
+        assertThat(suggestions).hasSize(7);
```

Comment:
I think it's better to just check if it's empty. Hardcoded size is a strange thing here

Reply:
right